lib/chart_style.dart

```diff
@@ -52,19 +52,14 @@ class ChartColors {
   Color minColor = Color(0xffffffff);
 
   Color getMAColor(int index) {
-    Color maColor = ma5Color;
     switch (index % 3) {
```

Comment:
% is a very slow operation, and the logic you modify is not equal to the previous one

Reply:
% was there before.
Please explain why logic is not equal? I removed the maColor var and added returns when case 1,2 or 0(default) comes. Same logic in my opinion.